Tasks/UseDotNetV2/task.json

```diff
@@ -14,7 +14,7 @@
     "version": {
         "Major": 2,
         "Minor": 0,
-        "Patch": 16
+        "Patch": 17
```

Comment:
Why update this task version if task lib is not updated

Reply:
dependent common/utility-common-2 has been changed

Reply:
Ohh yes